mirrormanager2/templates/fedora/site.html

```diff
@@ -1,120 +1,90 @@
 {% extends "master.html" %}
-{% from "_formhelpers.html" import render_field_in_row %}
+{% from "_macros.html" import site_form, export_compliance %}
+
 
 {% block title %}New Site{% endblock %}
 {%block tag %}home{% endblock %}
 
-{% block header %}
-<link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css"
-  rel="stylesheet">
-{% endblock %}
-
 {% block content %}
-<h2>Information site: {{ site.name }}</h2>
-
-Created at: {{ site.created_at }} <br />
-Created by: {{ site.created_by }}
-
-<h3>Export compliance</h3>
-<p>
-  By downloading Fedora software, you acknowledge that you understand all of
-  the following: Fedora software and technical information may be subject to
-  the U.S. Export Administration Regulations (the “EAR”) and other U.S. and
-  foreign laws and may not be exported, re-exported or transferred
-  (a) to any country listed in Country Group E:1 in Supplement No. 1 to part
-  740 of the EAR (currently, Cuba, Iran, North Korea, Sudan and Syria);
-  (b) to any prohibited destination or to any end user who has been prohibited
-  from participating in U.S. export transactions by any federal agency of the
-  U.S. government; or
-  (c) for use in connection with the design, development or production of
-  nuclear, chemical or biological weapons, or rocket systems, space launch
-  vehicles, or sounding rockets, or unmanned air vehicle systems.
-</p>
+<div class="mt-3 d-flex justify-content-between align-items-center">
+  <div>
+    <h2 class="mb-0"><span class="fa fa-globe"></span> {{ site.name }}</h2>
+  Created by <strong>{{ site.created_by }}</strong> at <strong>{{ site.created_at }}</strong>
```

Comment:
This looks a little bit weird:
![image](https://github.com/fedora-infra/mirrormanager2/assets/95780/e85224e8-9856-4c88-8616-b826d8f8a4ab)
Maybe do some formatting with the date? Nothing too fancy of course.

Reply:
Yeah -- i started that, but involved adding a filter -- so will merge this, and fix in another PR